### ISSUE TITLE
Displaying matches under Linked mentions?

### ISSUE THREAD

**Issue by crazymedguy** [SkepticMystic/graph-analysis#50]
So recently Live preview has enabled us to put by default, Linked mentions underneath the page. I was wondering whether it would be possible to put the graph analysis matches (limited by set number or pagination) to show under or above the Linked mentions area?

**Comment by HEmile** [comment]
@SkepticMystic you tried to implement something like this for Breadcrumbs right, how hard was it? I think you still have issues with scrolling right?

**Comment by SkepticMystic** [comment]
I think appending it to the bottom of the note will be easier. It wasn't difficult, just putting a Svelte component inside the right el

**Comment by HEmile** [comment]
Makes sense. I like the request, as it is in line with the backlinks panel. I suppose it mostly involves mounting the Svelte component in the right place, as you say.
